posthog/tasks/usage_report.py

```diff
@@ -662,6 +702,20 @@ def _get_all_usage_data(period_start: datetime, period_end: datetime) -> dict[st
     Gets all usage data for the specified period. Clickhouse is good at counting things so
     we count across all teams rather than doing it one by one
     """
+
+    integration_event_counts = get_teams_with_llm_integration_event_counts_in_period(period_start, period_end)
+
+    # Process the integration event counts
+    teams_with_event_count = {
+        "helicone": defaultdict(int),
+        "langfuse": defaultdict(int),
+        "keywords_ai": defaultdict(int),
+        "traceloop": defaultdict(int),
+    }
+
+    for team_id, integration, count in integration_event_counts:
+        teams_with_event_count[integration][team_id] = count
```

Comment:
I'd put this processing inside `get_teams_with_llm_integration_event_counts_in_period` so it's just called from the return statement like all the others.

Reply:
Oh, I see why you did this, so you aren't calling the function multiple times. I think `get_teams_with_llm_integration_event_counts_in_period` can return `teams_with_event_count["helicone"]` etc so you just call it once like you do up here, and then do `integration_event_counts["helicone"] in the return statement.

Reply:
Updated!